content-hour-row.php

```diff
@@ -66,136 +66,133 @@
 											<b>Today</b><br/>
 										<?php
 											$curDay = $now;
-											
-											$temp = $post;									
-											
-											//echo "S: ".date("Y-m-d", $today)." ";
-											
-											$message = getMessageDay($locationId, $today);
-											
-											if ($message != "" ) {
-												
-												if ($showMobileSpecial == 1) {
+
+											$temp = $post;
+
+											// echo "S: ".date("Y-m-d", $today)." "; .
```

Comment:
Yeah, I can see that argument. Would you prefer to tolerate getting dinged in a review for things like commented-out code, or would it be preferable to just remove the code altogether? Or would another path make more sense?